### ISSUE TITLE
Shipment item edit

### ISSUE THREAD

**Issue by IMiGS** [openboxes/openboxes#239]
When I try to remove items from shipped shipment (for example, courier took only 1 of two items) then I can successfully do it but items do not count back in my original warehouse and these counted items just disappear from inventory. The only solution is to click on the shipment and roll back till the shipment is in '"pending" status. It would be great if removed items from shipped shipment  went back to original inventory.

**Comment by jmiranda** [comment]
Yeah, I would not delete items from a shipped shipment. Can you describe why you would want to do that?

**Comment by IMiGS** [comment]
For example if shipment has been made but courier took only part of items and other items will be taken in another shipment

**Comment by jmiranda** [comment]
Ok, I'll think about this a bit more. It probably makes sense to handle cases like this, but we generally want to avoid editing shipments after they have been shipped.

Wouldn't you know that before you marked the shipment as shipped?

**Comment by IMiGS** [comment]
Ideally, we would send a pickup/shipment request through shipment form and at that moment we don't know which items will be pickup up.
For example, we set arrival and shipping date on the next week of the task creation date, we add all the items which will be shipped and send request/report copy to us and to the 3rd party which will handle the shipment. At that moment we don't now what will be picked up and we edit later when actual shipping date has come.
Maybe if the shipping date is in the future, then there must be status not Shipped but waiting pickup and actually it will be in pending status or some similar status when items can be removed from shipping tab  and they went back to original warehouse? But this would add additional step to confirm that items has been shipped.

**Comment by jmiranda** [comment]
Sorry meant to cancel my comment not close the ticket.
